__tests__/CSVExportActions.test.js

```diff
@@ -158,6 +158,12 @@ describe('csv export actions', () => {
           expect(value).toEqual(true);
           const dataPath = csvHelpers.dataPath(checksPerformedPath);
           const filePath = path.join(dataPath, 'output', 'Selections.csv');
+
+          // verify that gatewayLanguageQuote might exist if test files were created correctly
+          let csvData = fs.readFileSync(filePath, 'utf8' );
+          expect(csvData).toContain('klappy'); // Should be replaced with known quote. 
```

Comment:
Shouldn't this be looking for the quote? I think "klappy" is the username value.

Reply:
Addressed concern.